.eslintrc.json

```diff
@@ -1,3 +1,24 @@
 {
-  "extends": "next/core-web-vitals"
+  "extends": ["next/core-web-vitals", "marine/prettier/react"],
```

Comment:
never heard of marine/prettier/react - gonna have to do some reading on it!

Reply:
`eslint-config-marine` is the config itself, the `/prettier/react` is traversing files in it (to specifically use the prettier react variant)